common/shared/src/main/scala/com/wavesplatform/common/state/ByteStr.scala

```diff
@@ -7,14 +7,25 @@ import scala.util.Try
 case class ByteStr(arr: Array[Byte]) {
   lazy val base58: String = Base58.encode(arr)
 
-  lazy val base64: String = "base64:" + Base64.encode(arr)
-
-  lazy val trim: String = base58.toString.take(7) + "..."
-
-  override lazy val toString: String = base58
+  lazy val base64_row: String = Base64.encode(arr)
```

Comment:
please use `camelCaseNaming`, also you probably intended to say `raw`, not `row`